.github/workflows/ci.yml

```diff
@@ -39,6 +22,16 @@ jobs:
         name: Test
         with:
           command: test
+      - uses: actions-rs/cargo@v1
+        name: Format
+        with:
+          command: fmt
+          args: --all -- --check
+      - uses: actions-rs/cargo@v1
+        name: Clippy
+        with:
+          command: clippy
+          args: -- -D warnings
```

Comment:
These should be separate jobs to make so that they run in parallel giving the author quicker feedback

Reply:
Thanks for the explanation, I removed the splitting of Clippy, but kept cargo fmt in the main "Build" since that's basically instant. Would you like Format to be split too?